lib/Tag.js

```diff
@@ -116,6 +117,7 @@ Tag.propTypes = {
   connectDragSource: PropTypes.func.isRequired,
   isDragging: PropTypes.bool.isRequired,
   connectDropTarget: PropTypes.func.isRequired,
+  tagRenderer: PropTypes.func,
```

Comment:
can you move the proptypes and default props inside the class and make it static ?